lib/assets/stylesheets/flash.css.scss

```diff
@@ -13,14 +20,4 @@
   padding-top: 0;
 }
 
-.clickthrough {
```

Comment:
what about leaving this, for reference reasons? (I would like this and it would not take too much space, what do you think?)